pkg/latte/intent_login.go

```diff
@@ -148,7 +149,30 @@ func (i *IntentLogin) GetEffects(ctx context.Context, deps *workflow.Dependencie
 }
 
 func (i *IntentLogin) OutputData(ctx context.Context, deps *workflow.Dependencies, w *workflow.Workflow) (interface{}, error) {
-	return nil, nil
+	verifiedAuthns, err := i.getVerifiedAuthenticators(w)
+	if err != nil {
+		return nil, err
+	}
+	if len(verifiedAuthns) == 0 {
+		return nil, nil
+	}
+	userID := verifiedAuthns[0].UserID
+	identities, err := deps.Identities.ListByUser(userID)
```

Comment:
I think we can just get the standard attribute `email` and mask it, include it in the data as `masked_email`.

Reply:
I am not sure if it is safe to assume the email in standard attribute is same as the email identity. As forgot password only works for login ids, I get them from login ids.